ai-platform-unified/notebooks/official/explainable_ai/gapic-custom_image_classification_online_explain.ipynb

```diff
@@ -0,0 +1,2673 @@
+{
```

Comment:
<p>Can this be replaced with:</p><p> </p><p>REGION = '[your-region]'&nbsp;#@param {type: "string"}</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/352/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>